frontend/src/ui-components/AcmSearchInput/AcmSearchInput.tsx

```diff
@@ -53,7 +53,8 @@ export interface AcmSearchInputProps extends SearchInputProps {
   searchableColumns?: SearchableColumn[]
   canAddConstraints?: boolean
   disableAddConstraint?: boolean
-  pendingConstraints?: SearchConstraint[]
+  pendingConstraints: SearchConstraint[]
+  setPendingConstraints: (constraints: SearchConstraint[]) => void
```

Comment:
Control of this piece of state needed to be pulled out of the SearchInput, and controlled in the AcmTable component. Now the state and it's setter hook are handed back to the SearchInput as props.